builds/checkin/dotnet.yaml

```diff
@@ -63,11 +63,7 @@ jobs:
     pool:
       vmImage: "vs2017-win2016"
     steps:
-      - task: PowerShell@2
-        displayName: Install
-        inputs:
-          filePath: scripts/windows/setup/Install-Prerequisites.ps1
-          arguments: -DotnetSdkUrl $(NetCorePackageUriWindows) -Dotnet -Nuget
```

Comment:
we don't need NuGet anymore?

Reply:
Everything works fine without it. What was this used for? I assumed it was something we were no longer using like python.